packages/main/cypress/specs/ResponsivePopover.cy.tsx

```diff
@@ -0,0 +1,122 @@
+import Button from "../../src/Button.js";
+import ResponsivePopover from "../../src/ResponsivePopover.js";
+
+describe("ResponsivePopover general interaction", () => {
+	it("header and footer are displayed by default", () => {
+		cy.mount(
+			<>
+				<Button id="btnOpen" />
+				<ResponsivePopover id="respPopover" opener={"btnOpen"}>
+					<div slot="header" />
+					<div slot="footer" />
+				</ResponsivePopover>
+			</>
+		);
+
+		cy.get("#respPopover").invoke("attr", "open", true);
```

Comment:
there is custom command for responsive popover which checks whether the component is open and to open it